src/test/resources/cristina_savrin/The second Feature file.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Automation Task 2 - Using Python
+
+  Scenario: String length validation
+    Given I have a string "Java Help" CS
+    When Execute Python Script "len('{0}')" CS
+    Then Assert that "{1} == 9" CS
+
+  Scenario: String contains validation
+    Given I have a string "Java Help" CS
+    Then the string should contain "He" CS
```

Comment:
As you are using python I recommend to change this step to 
Assert that "'He' in '{0}'"